x-pack/plugins/observability_solution/infra/public/components/ml/anomaly_detection/anomalies_table/anomalies_table.tsx

```diff
@@ -148,7 +148,12 @@ const AnomalyActionMenu = ({
     };
 
     if (!hostName || !closeFlyout) {
-      hostLocator?.navigate({}).then(() => addHostFilter(newFilter));
+      hostLocator?.navigate({
+        dateRange: {
+          from: 'now-30d',
+          to: 'now',
```

Comment:
Users can change the date picker. Instead of `now-30d`, it could be that they want to see `now-60d`

<img width="865" alt="image" src="https://github.com/user-attachments/assets/c23cbe39-d15b-4ffe-8d47-d7109cc6f201">

what we ideally want to do is to propagate the value in the date picker to hostLocator

Reply:
Good point. 

BTW: This commit isn't ready. I pushed that for Jenny as a quick example because she runs my branch to help me :)

Reply:
Thanks for your input! I forgot that users can change value from dropdown 🙌

Reply:
We were checking different ways to pass params and apparently using `navigate` somehow was not working in this case so the parameters we passed were overwritten by the default values - probably it's related to some param validation or it's the same path and that somehow breaks it (it works when navigating from inventory). 
So one way to option that is to use `href` instead of  `hostsLocator?.getRedirectUrl`.